packages/app/src/app/components/SubscribeForm/CheckoutForm/elements.ts

```diff
@@ -1,12 +1,8 @@
 import styled from 'styled-components';
```

Comment:
Important to check if `/patron` and `/pro` changed styling after this change.

Reply:
Fixed and working